packages/components/src/mobile-promo-card/index.tsx

```diff
@@ -13,42 +14,110 @@ export type MobilePromoCardProps = {
 	isWoo?: boolean;
 };
 
+// Slugs as used by Jetpack Redirects.
+// See https://jetpack.com/redirect for current URLs.
+// Two _QRCODE constants are included for reference only.
+
+const URL_JETPACK_A8C = 'calypso-stats-mobile-cta-jetpack-link';
+const URL_JETPACK_APPLE = 'calypso-stats-mobile-cta-jetpack-apple-badge';
+const URL_JETPACK_GOOGLE = 'calypso-stats-mobile-cta-jetpack-google-badge';
+const URL_JETPACK_QRCODE = 'calypso-stats-mobile-cta-jetpack-qrcode';
+const URL_WOO_A8C = 'calypso-stats-mobile-cta-woo-link';
+const URL_WOO_APPLE = 'calypso-stats-mobile-cta-woo-apple-badge';
+const URL_WOO_GOOGLE = 'calypso-stats-mobile-cta-woo-google-badge';
+const URL_WOO_QRCODE = 'calypso-stats-mobile-cta-woo-qrcode';
+
+// Event names used for Tracks.
+const EVENT_JETPACK_VIEW = 'calypso_stats_mobile_cta_jetpack_view';
+const EVENT_JETPACK_CLICK = 'calypso_stats_mobile_cta_jetpack_click';
+const EVENT_JETPACK_CLICK_APPLE = 'calypso_stats_mobile_cta_jetpack_apple_click';
+const EVENT_JETPACK_CLICK_GOOGLE = 'calypso_stats_mobile_cta_jetpack_google_click';
+const EVENT_WOO_VIEW = 'calypso_stats_mobile_cta_woo_view';
+const EVENT_WOO_CLICK = 'calypso_stats_mobile_cta_woo_click';
+const EVENT_WOO_CLICK_APPLE = 'calypso_stats_mobile_cta_woo_apple_click';
+const EVENT_WOO_CLICK_GOOGLE = 'calypso_stats_mobile_cta_woo_google_click';
+
+// Maps the Jetpack Redirect slugs to URLs.
+// Named to match getRedirectUrl from '@automattic/jetpack-components'
+// which isn't available to us here.
+function getRedirectUrlPrivate( slug: string ) {
```

Comment:
Why is the function name suffixed with `Private`? Since the function's not being exported, its scope is already limited to this file.

Reply:
The goal with this is to help with readability in search results. I think the better option is probably to rename it to be slightly more specific though. So maybe something like `getRedirectUrlFromSlug()`. I'll give this another look.

Reply:
Addressed in #70926.